webdav3/client.py

```diff
@@ -235,18 +235,22 @@ def valid(self):
         return True if self.webdav.valid() else False
 
     @wrap_connection_error
-    def list(self, remote_path=root):
+    def list(self, remote_path=root, getinfo = False):
```

Comment:
@huangganggui Please use PEP8 for formatting:
- all parameters and variables should be under_scoped
- in optional parameters there is not spaces around '=', ex `get_info=False`.

Reply:
@huangganggui please add tests for that method in tests/test_client_it.py^ check there that the returned result contains all necessary parameters and does not contain parent folder (`remote_path`)